app/components/markdown/index.js

```diff
@@ -3,15 +3,17 @@
 
 import {Parser} from 'commonmark';
 import Renderer from 'commonmark-react-renderer';
-import React, {PureComponent} from 'react';
+import React from 'react';
```

Comment:
If this was giving you a `React` not used error/warning can you do `import {PureComponent} from 'react'` instead?